connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -817,19 +731,11 @@ public void testAccessors() throws Exception {
         Herder.Created<ConnectorInfo> connectorInfo = createCallback.get(1000L, TimeUnit.SECONDS);
         assertEquals(createdInfo(SourceSink.SOURCE), connectorInfo.result());
 
-        EasyMock.reset(transformer);
-        EasyMock.expect(transformer.transform(eq(CONNECTOR_NAME), EasyMock.anyObject()))
```

Comment:
Should we keep this assertion?

Reply:
@C0urante You have an opinion on this?

Reply:
I agree with @mimaison, we should keep this part. Otherwise, there's no reason to issue duplicate calls to `Herder::connectors`, `Herder::connectorInfo`, etc.

Reply:
Committed and pushed